olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PrefetchTilesRequest.h

```diff
@@ -65,64 +67,98 @@ class DATASERVICE_READ_API PrefetchTilesRequest final {
-   * @return a reference to the updated PrefetchTilesRequest.
+   * @brief Sets the tile keys for the request.
+   *
+   * @param tile_keys The rvalue reference to the vector with the tile keys.
+   *
+   * @return A reference to the updated `PrefetchTilesRequest` instance.
    */
   inline PrefetchTilesRequest& WithTileKeys(
       std::vector<geo::TileKey>&& tile_keys) {
     tile_keys_ = std::move(tile_keys);
     return *this;
   }
 
+  /**
+   * @brief Gets the minimum tile level.
+   *
+   * @return The minimum tile level.
+   */
   inline unsigned int GetMinLevel() const { return min_level_; }
+
+  /**
+   * @brief Sets the minimum tile level for the request.
+   *
+   * @param min_level The minimum tile level.
+   * @return A reference to the updated `PrefetchTilesRequest` instance.
+   */
   inline PrefetchTilesRequest& WithMinLevel(unsigned int min_level) {
     min_level_ = min_level;
     return *this;
   }
-
+  /**
+   * @brief Gets the maximum tile level.
+   *
+   * @return The maximum tile level.
+   */
   inline unsigned int GetMaxLevel() const { return max_level_; }
+
+  /**
+   * @brief Sets the maximum tile level for the request.
+   *
+   * @param max_level The maximum tile level.
+   * @return A reference to the updated `PrefetchTilesRequest` instance.
+   */
   inline PrefetchTilesRequest& WithMaxLevel(unsigned int max_level) {
     max_level_ = max_level;
     return *this;
   }
 
   /**
-   * @brief Sets the catalog version to be used for the requests.
+   * @brief Sets the catalog version for the request.
+   *
    * @param version The catalog version of the requested partitions. If no
-   * version is specified, the latest will be retrieved.
-   * @return a reference to the updated PrefetchTilesRequest.
+   * version is specified, the latest version is used.
+   *
+   * @return A reference to the updated `PrefetchTilesRequest` instance.
    */
   inline PrefetchTilesRequest& WithVersion(boost::optional<int64_t> version) {
     catalog_version_ = std::move(version);
     return *this;
   }
 
   /**
-   * @brief Get the catalog version requested for the partitions.
-   * @return The catalog version, or boost::none if not set.
+   * @brief Gets a catalog version for the request.
+   *
+   * @return The catalog version or `boost::none` if the catalog version is not
+   * set.
    */
   inline const boost::optional<std::int64_t>& GetVersion() const {
     return catalog_version_;
   }
 
   /**
-   * @brief BillingTag is an optional free-form tag which is used for
-   * grouping billing records together. If supplied, it must be between 4 - 16
-   * characters, contain only alpha/numeric ASCII characters  [A-Za-z0-9].
-   * @return the billing tag, or boost::none if not set.
+   * @brief Gets a billing tag to group billing records together.
+   *
+   * The billing tag is an optional free-form tag that is used for grouping
+   * billing records together. If supplied, it must be 4&ndash;16 characters
```

Comment:
```suggestion
   * billing records together. If supplied, it must be 4-16 characters
```
Would a real dash, i.e. `-`, be cleaner? The encoding is nice for HTML but it's harsh on humans:)

Reply:
Agree.
Added the en dash sign (–) instead of `&ndash;`.